owncloudApp/src/main/java/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java

```diff
@@ -960,6 +968,8 @@ public void afterTextChanged(Editable editable) {
                             MAX_FILENAME_LENGTH);
                 } else if (fileName.length() == 0) {
                     error = getString(R.string.uploader_upload_text_dialog_filename_error_empty);
+                } else if (fileName.contains("/")) {
+                   error = getString(R.string.filename_forbidden_characters);
```

Comment:
We introduced a [FileNameValidator on the new architecture](https://github.com/owncloud/android/blob/new_arch%2Fsynchronization/owncloudDomain/src/main/java/com/owncloud/android/domain/validator/FileNameValidator.kt) branch to handle every validation the same way. It is not available on master yet, so we will handle this when we merge/rebase with new architecture